doc/tutorials/using_celadon_as_uos.rst

```diff
@@ -0,0 +1,113 @@
+   to get the Celadon project source code ready.
+
+#. Select Celadon build target::
+
+      $ cd <Celadon project directory>
+      $ source build/envsetup.sh
+      $ lunch cel_apl-userdebug
+
+   .. note:: You can run ``lunch`` with no arguments to manually choose your Celadon build variants.
+
+#. Download these additional patches and apply each one individually with the command::
+
+       $ git apply <patch-filename>
+
+   .. table:: ACRN patch list
+      :widths: auto
+      :name: ACRN patch list
+
+      +--------------------------------------------------------------------+-------------------------------------------+
+      | Patch link                                                         | Description                               |
+      +====================================================================+===========================================+
+      | https://github.com/projectceladon/device-androidia/pull/458        | kernel config: Add the support of ACRN    |
+      +--------------------------------------------------------------------+-------------------------------------------+
+      | https://github.com/projectceladon/device-androidia-mixins/pull/293 | graphic/mesa: Add the support of ACRN     |
+      +--------------------------------------------------------------------+-------------------------------------------+
+      | https://github.com/projectceladon/device-androidia/pull/441        | cel_apl: use ttyS0 instead of ttyUSB0     |
+      +--------------------------------------------------------------------+-------------------------------------------+
+      | https://github.com/projectceladon/device-androidia/pull/439        | Disable trusty and pstore                 |
+      +--------------------------------------------------------------------+-------------------------------------------+
+
+   .. note:: If the ``git apply`` command shows an error, you may need to modify
+      the source code manually instead.
+
+#. Build Celadon image::
+
+   $ device/intel/mixins/mixin-update
+   $ make SPARSE_IMG=true gptimage -j $(nproc)
+
+.. note:: Replace the ``$(nproc)`` argument with the number of processor threads on your workstation
+   in order to build the source code with parallel tasks. The Celadon gptimage will be
+   generated to ``out/target/product/cel_apl/cel_apl_gptimage.img``
+
+Steps for Using Celadon as User OS
+**********************************
+
+#. Follow :ref:`getting-started-apl-nuc` to boot the "ACRN Service OS"
+
+#. Download the `OVMF.fd firmware image
+   <https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/tutorials/OVMF.fd>`_
+   from the ACRN GitHub repo to your development system.
+
+#. Download the `launch_android.sh script
+   <https://raw.githubusercontent.com/tanminger/acrn-hypervisor/ed7a758377151b4c7296c0b5669167585b50907b/devicemodel/samples/nuc/launch_android.sh>`_
+   to your development system.
```

Comment:
Why the link to a specific commit in someone's repo? I would add such file to our existing folder containing launch scripts here: https://github.com/projectacrn/acrn-hypervisor/tree/master/devicemodel/samples/nuc

Reply:
I will use ``launch_win.sh`` script instead since that's also functional.

Reply:
I have a slight preference for adding a new script called `launch_celadon.sh` (or `launch_android.sh`), we already have various scripts in there for different OSs and adding one more will bring less confusion than having to use one meant for Windows. (not mentionning the fact that if we later modify `launch_win.sh` I'm not sure we'll always remember to test those changes with Android/Celadon).